### PR TITLE
fix(fxa-payments-server): fix skipped paypal legal link

### PR DESCRIPTION
**Pull request by meandavejustice**
## This pull request

- We hadn't updated one of the paypal privacy links, this pr addresses that

## Issue that this pull request solves

Closes: #10164

## Checklist

_Put an `x` in the boxes that apply_

- [x] My commit is GPG signed.
- [ ] If applicable, I have modified or added tests which pass locally.
- [ ] I have added necessary documentation (if appropriate).
- [ ] I have verified that my changes render correctly in RTL (if appropriate).

### CODE REVIEW

File: packages/fxa-payments-server/src/components/PaymentLegalBlurb/index.tsx [hunk 1]
```diff
@@ -84,7 +84,7 @@ const DefaultPaymentLegalBlurb = () => (
         ),
         paypalPrivacyLink: (
           <a
-            href="https://paypal.com/privacy"
+            href="https://www.paypal.com/webapps/mpp/ua/privacy-full"
```
Comment (vbudhram) [inline packages/fxa-payments-server/src/components/PaymentLegalBlurb/index.tsx:87]: Looks like https://paypal.com/privacy redirects to ^. Maybe it is better to keep link as is?

Comment (chenba) [reply packages/fxa-payments-server/src/components/PaymentLegalBlurb/index.tsx:87]: > 
> 
> Looks like https://paypal.com/privacy redirects to ^. Maybe it is better to keep link as is?

Seconded.  https://paypal.com/privacy looks more like a permalink.

Comment (biancadanforth) [reply packages/fxa-payments-server/src/components/PaymentLegalBlurb/index.tsx:87]: It did one one day and didn't on another day more recently. This is the link [Tony asked us to use](https://mozilla-hub.atlassian.net/browse/FXA-3715?focusedCommentId=441990) in the ticket.

Comment (chenba) [reply packages/fxa-payments-server/src/components/PaymentLegalBlurb/index.tsx:87]: > 
> 
> It did one one day and didn't on another day more recently

LOL why is PayPal so terrible :'(

Comment (biancadanforth) [inline packages/fxa-payments-server/src/components/PaymentLegalBlurb/index.tsx:87]: I actually have no idea how this reverted back to https://paypal.com/privacy , as it was updated in #10074 ... (see [here](https://github.com/mozilla/fxa/pull/10074/files#diff-f8298171cc067c65c23e886c6d115f61da4000621aa7b060b8aa23dd425313dcL31)). @chenba any ideas?

My hypothesis is the localization ID. Can you change the localization ID here and in main.ftl to be safe?

**Review by biancadanforth — DISMISSED** [review]
Thanks Dave; can you update the localization ID for this link? I'm still not sure how it reverted from when you changed this in a [prior PR](https://github.com/mozilla/fxa/pull/10074/files#diff-f8298171cc067c65c23e886c6d115f61da4000621aa7b060b8aa23dd425313dcL31)...

File: packages/fxa-payments-server/src/components/PaymentLegalBlurb/index.tsx (hunk 1 above)
Comment (chenba) [reply packages/fxa-payments-server/src/components/PaymentLegalBlurb/index.tsx:87]: I was confused for a while, but that link is in two places.  This is not the same location as the one you linked in your comment.

Comment (biancadanforth) [reply packages/fxa-payments-server/src/components/PaymentLegalBlurb/index.tsx:87]: Oh right; same file but in two locations. Odd.

**Review by chenba — APPROVED** [review]

**Pull request merged**